lib/flutter_beacon.dart

```diff
@@ -212,6 +218,16 @@ class FlutterBeacon {
     return onRanging;
   }
 
+  /// Start monitoring ranging status.
+  ///
+  /// This will fires [RangingActivityResult] in every 5 seconds.
```

Comment:
can we ensure this happens every 4 seconds instead? I want to ensure we have sufficient buffer, when we resample it on the flutter app